chromium_src/chrome/common/pref_names.h

```diff
@@ -0,0 +1,17 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_COMMON_PREF_NAMES_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_COMMON_PREF_NAMES_H_
+
+#include "build/build_config.h"
+
+namespace prefs {
+extern const char kBraveVpnDnsConfig[];
```

Comment:
please add a comment explaining what this is for and what it contains

Reply:
done

Reply:
I think you must have changes you haven't pushed because there are things marked "done" where nothing has changed

Reply:
Pushed here https://github.com/brave/brave-core/pull/15989/commits/a09d001af25bd33f6a07edc39eaa581539179f75